packages/snap-preact-components/src/hooks/useA11y.tsx

```diff
@@ -1,13 +1,14 @@
 import { h } from 'preact';
-import { useEffect } from 'preact/hooks';
 
 export function useA11y(elem: any, tabIndex?: number, focusRingColor?: string) {
-	useEffect(() => {
+	const styleId = 'ssA11yFocusStyle';
```

Comment:
With the current setup it seems that only the first hook usage will create the style block and all other usages will utilize that color. If we want to keep the color as a parameter here we ought to make it work as expected.

I think we discussed potentially removing that parameter altogether and leveraging CSS to specify more specific color changes.